proposals/lip-0061.md

```diff
@@ -213,11 +233,12 @@ The certificate signature as byte array.
 ###### Execution
 
 ```python
-signCertificate(sk, networkIdentifier, c):
-    remove the aggregationBits and signature property from c
-    message = serialization of c according to LIP 0027
+def signCertificate(sk: bytes, chainID: bytes, c: Certificate) -> bytes:
+    c.aggregationBits = EMPTY_BYTES
+    c.signature = EMPTY_BYTES
+    message = encode(certificateSchema, c)
```

Comment:
We are changing the protocol here, as now the serialization still contains the empty field from the schema. I think this should be fine.

Reply:
This is the same comment as https://github.com/LiskHQ/lips/pull/184#discussion_r1026155967 below, I assume.